seamare/src/core_context.rs

```diff
@@ -0,0 +1,45 @@
+use super::error::{Result, SeamareError};
+use camino::Utf8Path;
+use guppy::graph::PackageGraph;
+use guppy::MetadataCommand;
+
+// TODO: maybe it's better to store `project_root` field like what x-linter does.
```

Comment:
I'm assuming this is fine the way you wrote it but cc @sunshowers in case they know of a reason we'd want to match what x-linter does.